agents/test/MarriageAgentE2E.java

```diff
@@ -0,0 +1,55 @@
+package grakn.simulation.agents.test;
+
+import grakn.client.GraknClient;
+import grakn.client.answer.Numeric;
+import graql.lang.Graql;
+import graql.lang.query.GraqlGet;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.function.Consumer;
+
+import static graql.lang.Graql.var;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class MarriageAgentE2E {
+
+    private GraknClient graknClient;
+
+    @Test
+    public void testMarriageAgentInsertsTheExpectedNumberOfMarriages() {
+        localhostGraknTx(tx -> {
+            GraqlGet.Aggregate marriagesCountQuery = Graql.match(
+                    var("m").isa("marriage").rel("marriage_husband", "husband").rel("marriage_wife", "wife")
+            ).get().count();
+
+            List<Numeric> answer = tx.execute(marriagesCountQuery);
+            int numMarriages = answer.get(0).number().intValue();
+            int expectedNumMarriages = 119;
+            assertThat(numMarriages, equalTo(expectedNumMarriages));
+        });
+    }
+
+    @Before
+    public void createClient() {
+        String host = "localhost:48555";
```

Comment:
When we're running the tests, the Grakn Server may not be sitting locally. We should pass a variable (of the Grakn URI) to the test upon running it. You can still have localhost as the fallback option if no argument is passed, like `Simulation.java:72`

Reply:
Very good point, change upcoming

Reply:
Fixed, now used as follows:
`bazel test //agents/test/... --test_arg=localhost:48555`
or omitting the argument to use the default
`bazel test //agents/test/...`